harper-core/src/linting/closed_compounds.rs

```diff
@@ -82,12 +82,94 @@ create_closed_compound_linter!(Northeastern, "north eastern", "northeastern");
 create_closed_compound_linter!(Upholstery, "up holstery", "upholstery");
 create_closed_compound_linter!(Proofread, "proof read", "proofread");
 
+create_closed_compound_linter!(Forever, "for ever", "forever");
+create_closed_compound_linter!(Into, "in to", "into");
+create_closed_compound_linter!(Everyone, "every one", "everyone");
+create_closed_compound_linter!(Nobody, "no body", "nobody");
+create_closed_compound_linter!(Somebody, "some body", "somebody");
+create_closed_compound_linter!(Anybody, "any body", "anybody");
+create_closed_compound_linter!(Nothing, "no thing", "nothing");
+create_closed_compound_linter!(Anyway, "any way", "anyway");
+create_closed_compound_linter!(Anytime, "any time", "anytime");
```

Comment:
"anytime" is the mistake, it's just become extremely common recently. "any time" is always correct.